README.md

```diff
@@ -18,6 +18,12 @@ Use one API thanks to Segment.io's [analytics.js](https://segment.com/docs/libra
 * Quantcast
 * Segment.io
 
+## Development & Testing
+
+NOTE: Any ad-blocking web-browser extension may block the entire "okgrow:analytics" package due to the word "analytics" being used in the package name. This only occurs when running Meteor in development mode as the files are not bundled. To work around this issue run your application in production mode (see below).
```

Comment:
I would maybe instead suggest that they disable their ad blocker for development (usually they ad blockers have white-list rules to allow this)